sklearn/decomposition/incremental_pca.py

```diff
@@ -166,6 +171,7 @@ def fit(self, X, y=None):
         self.singular_values_ = None
         self.explained_variance_ = None
         self.explained_variance_ratio_ = None
+        self.singular_values_ = None
```

Comment:
Not really, but the other attributes were set there, so I thought it would be good for consistency to have the singular values be set there as well.